qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/PersonSteps.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.hamcrest.core.Is.is;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.RestClient;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.AddPersonPage;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.ManagePersonPage;
+import org.openmrs.contrib.qaframework.page.PersonFormPage;
+
+public class PersonSteps extends Steps {
+	
+    private AdministrationPage administrationPage;
+    private ManagePersonPage managePersonPage;
+    private PersonFormPage personFormPage;
+    private AddPersonPage addPersonPage;
+    private String personUuid;
+    private TestData.PersonInfo personInfo;
+    private final String PERSON_NAME = RandomStringUtils.randomAlphabetic(8);
+    private final String PERSON_AGE = RandomStringUtils.randomNumeric(2);
+    private final String PERSON_FAMILY_NAME = RandomStringUtils.randomAlphanumeric(8);
+	
+    @Before(RunTest.HOOK.SELENIUM_PERSON)
+    public void setUp() {
+	initiateWithLogin();
+	personInfo = TestData.generateRandomPerson();
+	personUuid = TestData.createPerson(personInfo);
+    }
+	
+    @After(RunTest.HOOK.SELENIUM_PERSON)
+    public void teardown() {
+	RestClient.delete("person/" + personUuid, true);
+	quit();
+    }
+	
+    @Given ("a user clicks on the system administration link on the home page")
+    public void launchSystemAdministrationPage() {
+    	administrationPage = homePage.goToAdministration();
+    }
+    
+    @Then ("the system loads the administration page")
+    public void loadAdministrationPage() {
+    	assertTrue(textExists("Administration"));
+    }
+    
+    @And ("a user clicks on the manage Persons link on the administration page")
+    public void launchAdvancedAdministrationPage() {
```

Comment:
take a close look at the method name and find out whether it reflects what the method is doing.